src/engine/Input/Pointer.ts

```diff
@@ -80,6 +86,44 @@ export class PointerEvent extends GameEvent<any> {
 };
 
 /**
+ * Wheel Events
+ *
+ * Represents a mouse wheel event. See [[Pointers]] for more information on
+ * handling point input.
+ */
+export class WheelEvent extends GameEvent<any> {
```

Comment:
Thinking about it, maybe the Enum should be:

```ts
enum WheelDirection {
 Forward,
 Backward,
 Left,
 Right,
 Inward,
 Outward
}
```

How does that sound @kamranayub ?